### ISSUE TITLE
Current SpaceAfterPropertyColon linter allows newlines

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#434]
While writing a PR for #428 I found that it currently only counts literal space characters:

x.scss

```scss
p {
  height: 0;
  margin:0;
  padding: 
  0;
  width:     0;
}
```

```bash
$ bundle exec bin/scss-lint x.scss
x.scss:3 [W] SpaceAfterPropertyColon: Colon after property should be followed by one space
x.scss:6 [W] SpaceAfterPropertyColon: Colon after property should be followed by one space
```

(You can't tell, but there is a ` ` after `padding:` (highlight it with the cursor).) So `padding` gets through the linter, because the series of space characters after the colon is exactly 1 space long (before it hits a different character, the newline character).

I assume this is unintentional? I can fix this before fixing #428 if it is. The fix would make it so that the following hold:

Let `\s` represent a space, `\t` represent a tab, and `\n` represent a newline. The `monospaced` text below are characters between a PROPERTY and its VALUE.

* When `style` is the default `one_space`:
  * The following PASS: `:\s`.
  * The following are LINT: `:`, `:\t`, `:\n`, `:\s\s`, `:\s\n`, `:\n\s`
* When `style` is `at_least_one_space`:
  * The following PASS: `:\s`, `:\s\s`, `:\s\s\s`.
  * The following are LINT: `:`, `:\t`, `:\n`, `:\s\n`, `:\n\s`, `:\s\t`, `:\t\s`

**Comment by sds** [comment]
The behavior described does seem unintentional. The proposal to address it seems reasonable. :+1:
